lib/hermann/consumer.rb

```diff
@@ -24,6 +24,10 @@ class Consumer
     #
     def initialize(topic, opts = {})
       @topic = topic
+
+      offset = opts.delete(:offset)
+      raise "Bad offset: #{offset}" unless valid_offset?(offset)
```

Comment:
Would you mind typing this exception and defining it in `lib/hermann/errors.rb`?